pkg/healthcheck/healthcheck.go

```diff
@@ -394,6 +394,7 @@ type Options struct {
 	DataPlaneNamespace    string
 	KubeConfig            string
 	KubeContext           string
+	VizNamespaceOverride  string
```

Comment:
Rather than adding a viz-specific option into the base Options struct here, we should keep all viz-specific code in the viz directory.  In this case, I think that means having the viz healthcheck package define a new `VizOptions` struct which has this `Options` struct as an embedded struct but also adds a new `VizNamespaceOverride` field.

Reply:
I hope the new changes are what you suggested :)